lib/generators/rspec/install/templates/spec/rails_helper.rb

```diff
@@ -34,7 +34,7 @@
 RSpec.configure do |config|
 <% if RSpec::Rails::FeatureCheck.has_active_record? -%>
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
-  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  config.fixture_path = Rails.root.join('spec/fixtures')
```

Comment:
```suggestion
  config.fixture_path = Rails.root.join('spec', 'fixtures')
```
To support those platforms with the `\` file separator.